src/main/resources/avro/metadata.avdl

```diff
@@ -9,7 +9,7 @@ protocol Metadata {
 import idl "common.avdl";
 
 /**
-An experimental preparation of a `Sample`.
+An experimental preparation of a `Biosample`.
 */
```

Comment:
Are these Biosample comments intentional? We don't have this defined, so perhaps it's better to revert these to Sample (even though that's not defined either!).